starfish/intensity_table/intensity_table_coordinates.py

```diff
@@ -20,30 +20,21 @@ def transfer_physical_coords_from_imagestack_to_intensity_table(image_stack: Ima
     """
     # TODO shanaxel42 consider refactoring pixel case where were can just reshape from Imagestack
     # Add three new coords to xarray (xc, yc, zc)
-    num_features = intensity_table.sizes[Features.AXIS]
-    intensity_table[Coordinates.X.value] = xr.DataArray(np.zeros(num_features, np.float32),
-                                                        dims='features')
-    intensity_table[Coordinates.Y.value] = xr.DataArray(np.zeros(num_features, np.float32),
-                                                        dims='features')
-    intensity_table[Coordinates.Z.value] = xr.DataArray(np.zeros(num_features, np.float32),
-                                                        dims='features')
-    for ind, spot in intensity_table.groupby(Features.AXIS):
```

Comment:
huh. i wouldn't have thought this would be so much slower than what you wrote below.